### ISSUE TITLE
Multiple object detection in Raster Vision

### ISSUE THREAD

**Issue by sagar1899** [azavea/raster-vision#1170]
How can i detect multiple object detection with different class id in geojson file.

Suppose there are 5 objects in training datasets.
in geojson file attribute id is "object_type" and attribute value is car, tree, building, road, hospitals.

so i want output with different attribute value of respective object type.

suppose there was car in output image, so in geojson file attribute value of "object_type" is "car",
suppose there was tree in output image, so in geojson file attribute value of "object_type" is "tree",
and so on...

**Comment by AdeelH** [comment]
Looking at the relevant code, https://github.com/azavea/raster-vision/blob/master/rastervision_core/rastervision/core/data/vector_source/class_inference.py#L24-L56, it appears that RV expects the class names to be contained in a property called `"class_name"` or `"label"`. So, if you were to rename your "object_type"` attribute to one of those two, it should be able to read in those names.

You would then need to modify the class config (https://github.com/azavea/raster-vision/blob/master/rastervision_pytorch_backend/rastervision/pytorch_backend/examples/object_detection/cowc_potsdam.py#L86) to include the names of all your classes like so: `ClassConfig(names=['car', 'tree', ...])`.
